.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
- [ ] I kindly suggest that you add a '/' in front of `node_modules` to indicate that it's a folder.